README.md

```diff
@@ -272,6 +278,21 @@ Perform multiple _put_ and/or _del_ operations in bulk. The `operations` argumen
 
 Each operation must be an object with at least a `type` property set to either `'put'` or `'del'`. If the `type` is `'put'`, the operation must have `key` and `value` properties. It may optionally have `keyEncoding` and / or `valueEncoding` properties to encode keys or values with a custom encoding for just that operation. If the `type` is `'del'`, the operation must have a `key` property and may optionally have a `keyEncoding` property.
 
+An operation of either type may also have a `sublevel` property, to prefix the key of the operation with the prefix of that sublevel. This allows atomically committing data to multiple sublevels. Keys and values will be encoded by the sublevel, to the same effect as a `sublevel.batch(..)` call. In the following contrived example, the `value` will be encoded with `'json'` rather than the default encoding of `db`:
+
+```js
+const people = db.sublevel('people', { valueEncoding: 'json' })
+
+await db.batch([{
+  type: 'put',
+  sublevel: people,
+  key: '123',
+  value: {
+    name: 'Alice'
+  }
+}])
+```
```

Comment:
I think since this option is primarily for committing to multiple sublevels, we should also show that use case. In this example I would wonder why one doesn't write `people.batch`

Reply:
Yeah, that's why I called it a "contrived example". Wanted to keep the example short, it's just to explain the encodings. But adding one other operation will just be a few lines, so I'll add it.